### PR TITLE
[CoreBundle, OrderBundle] display creation date in voucher list

### PR DESCRIPTION
**Pull request by solverat**
see #245.

### CODE REVIEW

File: src/CoreShop/Bundle/CoreBundle/Migrations/Version20171220075432.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20171220075432 extends AbstractPimcoreMigration implements ContainerAwareInterface
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Migrations/Version20171220075432.php]: Why do we need a migration here? Nothing on shared translations has been changed

Comment (solverat) [reply src/CoreShop/Bundle/CoreBundle/Migrations/Version20171220075432.php]: yea, but some commits before..

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Migrations/Version20171220075432.php]: fair enough :)

**Comment by dpfaffenbauer** [comment]
cheers

**Pull request merged**